tests/integration/search.test.ts

```diff
@@ -17,32 +17,38 @@ describe('Searching', function () {
   const client = new MongoClient(connectionString);
   let index: SearchIndex;
 
-  before('Loading test data', async function () {
-    await client.connect();
-    index = new SearchIndex(
-      'dir:tests/integration/search_test_data/',
-      'docs-search-indexes-test',
-      'search-indexes/preprd',
-      client,
-      TEST_DATABASE
-    );
-    const result = await index.load({} as Taxonomy, 'dir:tests/integration/search_test_data/');
-    await index.createRecommendedIndexes();
-
-    // I don't see a way to wait for indexing to complete, so... just sleep for some unscientific amount of time 🙃
-    if (result && (result.deleted || result.updated.length > 0)) {
-      this.timeout(8000);
-      await new Promise((resolve) => setTimeout(resolve, 5000));
+  this.beforeAll('Loading test data', async function () {
+    try {
+      await client.connect();
+      index = new SearchIndex(
+        'dir:tests/integration/search_test_data/',
+        'docs-search-indexes-test',
+        'search-indexes/preprd',
+        client,
+        TEST_DATABASE
+      );
+      const result = await index.load({} as Taxonomy, 'dir:tests/integration/search_test_data/');
+      console.log('index loaded');
+      await index.createRecommendedIndexes();
+      console.log('created recommended indexes');
+      console.log(result);
```

Comment:
(Just wanted to make sure) Are we purposely leaving these console logs?

Reply:
was debugging some issues with timeout and createIndex responding without actually creating the index (a 200 response means the request was good, not that the index has finished building).  think it'll be good to leave this for now to debug any flaky tests